examples/dgmulti_1d/elixir_euler_flux_diff.jl

```diff
@@ -11,9 +11,8 @@ equations = CompressibleEulerEquations1D(1.4)
 initial_condition = initial_condition_convergence_test
 source_terms = source_terms_convergence_test
 
-cells_per_dimension = (8, )
-vertex_coordinates, EToV = StartUpDG.uniform_mesh(dg.basis.elementType, cells_per_dimension...)
-mesh = VertexMappedMesh(vertex_coordinates, EToV, dg, is_periodic=(true,))
+mesh = DGMultiMesh(dg, cells_per_dimension=(8,),
+                   coordinates_min=(-1, -1), coordinates_max=(1, 1), periodicity=(true,))
```

Comment:
```suggestion
                   coordinates_min=(-1.0, -1.0), coordinates_max=(1.0, 1.0), periodicity=(true,))
```
Please also check all other modified elixirs

Reply:
Done